fabtests/pytest/common.py

```diff
@@ -0,0 +1,241 @@
+import pytest
+
+
+def check_returncode(returncode, strict):
+    import errno
+
+    if returncode == 0:
+        return
+
+    if not strict:
+        if returncode == errno.ENODATA:
+            pytest.skip("ENODATA")
+            return
+
+        if returncode == errno.ENOSYS:
+            pytest.skip("ENOSYS")
+            return
+
+    error_msg = "returncode {}".format(returncode)
+    if returncode == 124:
```

Comment:
would be helpful to add a comment about the exit value of the `timeout` command.

Reply:
Added a comment in new revision.